test_suite/testdata/go/dict.star

```diff
@@ -88,9 +88,11 @@ setIndex(x9, [], 2) ### (unhashable|not hashable)
 ---
 
 x9a = {}
-x9a[1, 2] = 3  # unparenthesized tuple is allowed here
+x9a[1, 2] = 3  ### rust: TODO(stepancheg): checked incorrectly on rust because error message includes snippet
```

Comment:
is there a part of the error message that's usable?

Reply:
As I mentioned in another part of this PR, we just need to get rid of `###` and the following text before evaluating starlark, so this string won't be in error message. Everything we put here will be in error message.